app/controllers/photo_likes_controller.rb

```diff
@@ -0,0 +1,14 @@
+class PhotoLikesController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+    photo = Photo.find(params[:photo_id])
+    photo_like = PhotoLike.new
+    photo_like.photo = photo
+    photo_like.user = current_user
+    unless photo_like.save
```

Comment:
Style/IfUnlessModifier: Favor modifier `unless` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`. (https://github.com/bbatsov/ruby-style-guide#if-as-a-modifier)